sdcm/sct_events/operator.py

```diff
@@ -28,74 +28,49 @@ class ScyllaOperatorLogEvent(LogEvent):
     REAPPLY: Type[LogEventProtocol]
     TLS_HANDSHAKE_ERROR: Type[LogEventProtocol]
     OPERATOR_STARTED_INFO: Type[LogEventProtocol]
-    timestamp: str
-    namespace: str
-    cluster: str
     message: str
-    error: str
-    trace_id: str
-
-    event_data_mapping = {
-        'T': 'timestamp',
-        'N': 'namespace',
-        'M': 'message',
-        'cluster': 'cluster',
-        'error': 'error',
-        '_trace_id': 'trace_id'
-    }
-
-    # pylint: disable=too-many-arguments
-    def __init__(self, timestamp=None, namespace=None, cluster=None, message=None, error=None, trace_id=None,
-                 regex=None, severity=Severity.ERROR):
-        super().__init__(regex, severity=severity)
-        self.set_params(timestamp=timestamp, namespace=namespace, cluster=cluster, message=message, error=error,
-                        trace_id=trace_id)
-
-    @property
-    def msgfmt(self):
-        cluster = f"/{self.cluster}" if self.cluster else ""
-        return super().msgfmt + " {0.trace_id} {0.namespace}" + cluster + ": {0.message}, {0.error}"
+    source: str
 
-    # pylint: disable=too-many-arguments
-    def set_params(self, timestamp=None, namespace=None, cluster=None, message=None, error=None, trace_id=None):
-        self.namespace = namespace
-        self.cluster = cluster
+    def __init__(self, message: str = '', source: str = '', regex: str = None, severity=Severity.NORMAL):
+        super().__init__(regex=regex, severity=severity)
         self.message = message
-        self.timestamp = timestamp
-        self.error = error
-        self.trace_id = trace_id
-
-    def load_from_json_string(self: T_log_event, data: str) -> Optional[T_log_event]:
-        try:
-            log_data = json.loads(data)
-        except Exception as exc:  # pylint: disable=broad-except
-            LOGGER.error('Failed to interpret following log line:\n%s\n, due to the: %s', data, exc)
-            return None
-        event_data = {}
-        for log_en_name, attr_name in self.event_data_mapping.items():
-            log_en_data = log_data.get(log_en_name, None)
-            if log_en_data is None:
-                continue
-            if attr_name == 'timestamp':
-                log_en_data = datetime.strptime(log_en_data, "%Y-%m-%dT%H:%M:%S.%fZ").timestamp()
-            event_data[attr_name] = log_en_data
+        self.source = source
+        self.line_number = None
+
+    # pylint: disable=too-many-locals
+    def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_event:
+        # I0628 15:53:02.269804       1 operator/operator.go:133] <message>
+        splits = line.split(maxsplit=4)
+        if len(splits) != 5 or len(splits[0]) != 5:
+            return self
+        type_month_date, time_string, _, self.source, self.message = splits
```

Comment:
Do we need to analyze "I0628" and fetch the severity of the line (Info, Error, for example)?

Reply:
Nope, severity should go from the pattern, not from the log message